### ISSUE TITLE
Multiple calls to a method that uses HttpClient.send causes MaxListenersExceededWarning (Node.js 19.1.0)

### ISSUE THREAD

**Issue by dwirz** [firebase/firebase-admin-node#1995]
### Environment

  * Operating System version: alpine 3.16
  * Firebase SDK version: 11.3.0
  * Firebase Product: auth (but potentially others as well)
  * Node.js version: 19.1.0
  * NPM version: 8.19.3

#### Steps to reproduce:

1. Call for example `Auth.getUser()` multiple times
2. Node.js logs a `MaxListenersExceededWarning` message

#### Relevant Code:

```
const auth = getAuth();
for (let index = 0; index < 20; index++) {
  await auth.getUser('some-id');
}
```

#### Additional Information

see #1993

**Comment by lahirumaramba** [comment]
Thank you @dwirz for flagging this issue and thank you for your PR! Let's discuss on the PR thread.

**Comment by lahirumaramba** [comment]
Fixed in #1993 
Thank you!
